myscripts.js

```diff
@@ -0,0 +1,24 @@
+const menuBtn = document.querySelector('.openNav');
+const closeBtn = document.querySelector('.menu-list');
+const contactLink = document.querySelector('.contact-link');
+
+menuBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  document.getElementById('myLinks').style.width = '100%';
+  document.getElementById('myLinks').style.display = 'block';
+});
+
+closeBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  document.getElementById('myLinks').style.width = '0%';
+});
+
+portfolioLink.addEventListener('click', () => {
+  document.getElementById('myLinks').style.width = '0%';
+});
```

Comment:
- To ensure everything runs smoothly, it would be great if you could define the variable `portfolioLink`. This will help us avoid any potential issues. 😊

- According to the project requirements, when a user clicks (or taps) any option of the mobile menu, the mobile menu should disappear.